raiden_libs/messages/__init__.py

```diff
@@ -0,0 +1,7 @@
+from .message import Message
+from .balance_proof import BalanceProof
+
+__all__ = [
+    Message,
```

Comment:
I always warning that these should be strings from mypy. See https://docs.python.org/2/tutorial/modules.html#importing-from-a-package

Reply:
ah, thanks. I wonder why does it still work if __all__ content is supposed to be a list of strings.

Reply:
Hmm but does it actually work? Have you tried to do `from messages import *` anywhere?
The `__all__` is supposed to be a list of names(strings), not values so it should throw a Type exception when the list does not contain strings.

Reply:
aha, that's why. I thought that `__all__` also works for direct imports (`from a.b.c import x`). I rarely do `*` imports so I haven't triggered the exception.